platformcode/updater.py

```diff
@@ -23,7 +23,7 @@
 _hdr_pat = re.compile("^@@ -(\d+),?(\d+)? \+(\d+),?(\d+)? @@.*")
 
 branch = 'master'
-user = 'kodiondemand'
+user = 'aandroide'
```

Comment:
Immagino fatta per comodità, quando sei pronto rimetti kodiondemand

Reply:
Ciao, si esatto sto testando in locale su PC e il mio GitHub era solo per comodità